service/src/main/resources/application.yml

```diff
@@ -1,6 +1,5 @@
 logging:
   level.bio.terra.cbas: ${BATCHANALYSIS_LOG_LEVEL:info}
-  pattern.level: '%X{requestId} %5p'
```

Comment:
This was being overriden by terra-common-lib configuration anyways, and is now replaced by `logback.xml`